resources/js/confirm-delete.js

```diff
@@ -1,5 +1,5 @@
 document.body.addEventListener("submit", function(event) {
-  if (event.target.querySelector('input[name="_method"][value="delete"]')) {
+  if (event.target.querySelector('input[name="_method"][value="delete"], input[name="_method"][value="DELETE"]')) {
```

Comment:
As an alternative we could perhaps use the case-insensitive selector, i.e. doing something like this:

`if (event.target.querySelector('input[name="_method"][value="delete" i]`

Here's the browser support for it:

https://www.w3cschool.cn/doc_browser_support_tables/browser_support_tables-css-case-insensitive.html

What do you think?

Reply:
Good idea, but sorry, that support table is a bit weird. This one shows no support in either IE or Edge: https://caniuse.com/#feat=css-case-insensitive